sixtyfps_runtime/interpreter/eval.rs

```diff
@@ -180,7 +180,7 @@ pub fn eval_expression(expression: &Expression, local_context: &mut EvalLocalCon
                 (Value::Model(model), Value::Number(index)) => {
                     if (index as usize) < model.row_count() {
                         model.model_tracker().track_row_data_changes(index as usize);
-                        model.row_data(index as usize)
+                        model.row_data(index as usize).unwrap_or(Value::Void)
```

Comment:
Not 100% sure this suggestion compiles, but basically I think we shouldn't return `Void` here.

```suggestion
                        model.row_data(index as usize).unwrap_or(default_value_for_type(expression.ty()))
```

Reply:
That makes only sense if I also replace the Value::Void two lines down. This unwrap should never trigger anyway as we have a valid row_count here.

Reply:
Indeed, so `unwrap()` is probably the right choice then. If the model decides to return `None` despite an in-range row index, then we want the panic to happen right here and not later when we encounter `Value::Void` but expected something else.